bin/Cargo.toml

```diff
@@ -126,6 +126,7 @@ tracing-subscriber = { workspace = true, features = [
 tokio = { workspace = true, features = ["time", "rt"] }
 hickory-client.workspace = true
 hickory-proto.workspace = true
+hickory-resolver.workspace = true
```

Comment:
It doesn't make sense to me that we're adding a dependency here for this. Do we have an indirect dependency on the resolver crate through some other path?

Reply:
We do: by activating the `resolver` feature, `hickory-resolver` gets transitively pulled in by `hickory-server`.

Reply:
(I think) eeeacd4 will make the dependency optional.

Reply:
If the resolver is a transitive dependency via the server crate, we should forward the features to the server crate (and from there to the resolver crate, if necessary).

Reply:
Good point, done.